### PR TITLE
Explain how to run only one test

### PR DESCRIPTION
**Pull request by aaronang**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| Breaking change?  |no
| New feature?      | no
| Deprecations?     | no
| Spec compliancy?  | no
| Tests added/pass? | no
| Fixed tickets     | 
| License           | MIT

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,6 +53,28 @@ descriptive name, and add the following:
...
+}
+```
+
+To run the test:
+
+```bash
+$ npm run test-only
+```
+
+
```
Comment (hzoo) [inline CONTRIBUTING.md]: Unnecessary whitespace?

**Comment by hzoo** [comment]
Maybe just move the section closer to the other running tests part?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,6 +31,26 @@ npm run test-only
 Note, this does not actually run a build, so you may have to call `npm run build` after
 performing any changes.
 
+### Running one test
+
+To run only one test of interest, you can add `"only": true` to `options.json` of any test.
+For illustration purposes, assume we would like to run the test [`test/fixtures/comments/basic/shebang-import`](https://github.com/babel/babylon/tree/7.0/test/fixtures/comments/basic/shebang-import).
```
Comment (xtuc) [inline CONTRIBUTING.md]: I would add a new line here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -31,6 +31,26 @@ npm run test-only
 Note, this does not actually run a build, so you may have to call `npm run build` after
 performing any changes.
 
+### Running one test
+
+To run only one test of interest, you can add `"only": true` to `options.json` of any test.
```
Comment (existentialism) [inline CONTRIBUTING.md]: Maybe something like:

> To run only a single test, add `"only": true` to the `options.json` inside any test fixture folder (you may have to create the file if it doesn't exist).

File: CONTRIBUTING.md (hunk 2 above)
Comment (existentialism) [inline CONTRIBUTING.md]: For example, let's say we want to only run the test for the [`test/fixtures/comments/basic/shebang-import`]() fixture.

Add the `only` option to its [`options.json`](https://github.com/babel/babylon/blob/7.0/test/fixtures/comments/basic/shebang-import/options.json):

```json
{
  "sourceType": "module",
  "only": true
}
```

Then, run the tests using the same command as before:

```sh
$ npm run test-only
  ```

**Comment by aaronang** [comment]
Thank you for the feedback! 

@existentialism Thanks for rewriting everything 😂
